src/client.js

```diff
@@ -47,7 +47,13 @@ function generateIrx() {
 function login() {
     let key = utils.sanitizeString(process.env.INPUT_ASOC_KEY);
     let secret = utils.sanitizeString(process.env.INPUT_ASOC_SECRET);
-    return executeCommand(`api_login -u ${key} -P ${secret} `);
+    if (process.env.INPUT_SERVICE_URL != null) {
+	    let service_url = utils.sanitizeString(process.env.INPUT_SERVICE_URL);
+	    return executeCommand(`api_login -u ${key} -P ${secret} -service_url ${service_url} -acceptssl`);
```

Comment:
We don't want to default to using -acceptssl. We only want to use that for specific cases where the user is explicitly choosing to make an insecure connection. That would need to be added as a separate option that would also need to be used for other commands (e.g. queue_analysis).

Reply:
Done.

Reply:
Ah right. I was thinking this was automated and we wouldn't get a prompt, but that's right. This would work similarly to how some of our other client components work where they need to add the acceptssl manually.

Reply:
So I think after the user logs in, we are reading from the cli.token, right? If so, then we don't need to add -acceptssl to the queue_analysis? Similar to the checking status.